.github/workflows/test.yml

```diff
@@ -21,8 +21,9 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+          allow-prereleases: true
           cache: pip
-          cache-dependency-path: setup.py
+          cache-dependency-path: pyproject.toml
```

Comment:
`setup.py` is just a single line that does not change (and can be eliminated -- #158) while `pyproject.toml` contains the list of dependencies that should be our trigger for dependency cache invalidation.